atom/browser/mac/atom_application_delegate.mm

```diff
@@ -52,12 +52,8 @@ - (NSApplicationTerminateReply)applicationShouldTerminate:(NSApplication*)sender
 - (BOOL)applicationShouldHandleReopen:(NSApplication*)theApplication
                     hasVisibleWindows:(BOOL)flag {
```

Comment:
Any reason why it was implemented with `applicationShouldHandleReopen ` instead of `applicationDidBecomeActive` ?